test/e2e/tests/signature-request.spec.js

```diff
@@ -126,7 +126,7 @@ describe('Sign Typed Data V3 Signature Request', function () {
         );
         const origin = content[0];
         const address = content[1];
-        const message = await driver.findElement(
+        const message = await driver.findElements(
```

Comment:
Should this be `messages` since there are now more than one?

Reply:
Updated